src/simulateApp/validation.js

```diff
@@ -0,0 +1,5 @@
+import { body } from 'express-validator';
+
+export const validatePassengers = () => [
+	body('passengers', 'This should be a number').optional().toInt(),
+];
```

Comment:
What will happen if a driver entered like 1 million as the total seats a car can hold? 
or Zero seats???
ie if you can just add validations for that...

Reply:
Thank you for mentioning @Abdulkeza 
I have added the validation to ensure that we are not exceeding the number of seats